src/codegen/compile.ml

```diff
@@ -172,7 +171,7 @@ module SR = struct
 
   let unit = UnboxedTuple 0
 
-  let bool = UnboxedBool
+  let bool = Vanilla
```

Comment:
Can we zap this binding now?

Reply:
Yes, I'd say so

Reply:
I'll leave it in, as it is helpful documentation that this is the {`0`, `1`} subset of `Vanilla`.